cmd/create.go

```diff
@@ -161,7 +167,7 @@ A full list of component types that can be deployed is available using: 'odo cat
 		} else if len(componentLocal) != 0 {
 
 			// Use the absolute path for the component
-			dir, err := filepath.Abs(componentLocal)
+			dir, err := util.GetAbsPath(componentLocal)
```

Comment:
Because `~` is only resolvable by shell and not resolved by any golang lib...
So, we'll need to work around it be resolving it to its absolute path..

Reply:
I also have a question as in https://github.com/redhat-developer/odo/pull/981#issuecomment-438757975 :smile:

Reply:
but yeah it doesn't make sense to be doing samething twice so I'll fix it in my pr